Add okbchain testnet and mainnet

ilovers opened the pull request:
The testnet and mainnet are not yet online, we just want to keep the chain id first.

ligi reviewed (changes requested):
RPC chainId (65) does not match chainId from json (195)

ilovers commented:
> RPC chainId (65) does not match chainId from json (195)
Updated. Our new chain is not launched and i have added the status to incubating in the json file.  I will update these json files when the chain is launched. Thanks!
